components/common/src/ome/api/Search.java

```diff
@@ -158,7 +158,7 @@
 
     /**
      * Returns the current case sensitivity setting. If
-     * {@link #setCaseSentivice()} has not been called, the
+     * {@link #setCaseSentivice(boolean)} has not been called, the
```

Comment:
you will need to indicate that the existing method will be deprecated before you can remove it
in that case a PR in 5.1 to mention it